src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { Header } from './Header';
+
+afterEach(cleanup);
+
+const setup = () => {
+  const history = createMemoryHistory();
+  const utils = render(
+    <Router history={history}>
+      <Header />
+    </Router>
+  );
+
+  const searchInput = utils.getByTestId('searchInput') as HTMLInputElement;
+  const headerLinks = utils.getAllByRole('link');
+  const dropdownOpener = utils.getByTestId('dropdownOpener') as HTMLButtonElement;
+
+  return {
+    utils,
+    searchInput,
+    headerLinks,
+    dropdownOpener
+  };
+};
+
+test('is serach render in Header', async () => {
```

Comment:
describe is missing (look at other files) and it

describe text should 

describe("<Header />")